packages/server-side-render/package.json

```
@@ -0,0 +1,36 @@
+{
+	"name": "@wordpress/server-side-render",
+	"version": "1.0.0-alpha.1",
+	"description": "ServerSideRender is a component used for server-side rendering a preview of dynamic blocks to display in the editor.",
```

How about we make it more explicit that this component won't work outside of WordPress?
```suggestion
	"description": "The component used with WordPress to server-side render a preview of dynamic blocks to display in the editor.",
```